src/components/Form/Radiobutton.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { Component, ReactNode } from 'react';
+import classnames from 'classnames';
+import { HtmlInput, HtmlLabel, HtmlSpan, HtmlDiv } from '../../reset';
+import { idGenerator } from '../../utils/uuid';
+import { logger } from '../../utils/logger';
+import {
+  RadiobuttonGroupConsumer,
+  RadiobuttonGroupProviderState,
+} from './RadiobuttonGroup';
+
+const baseClassName = 'fi-radiobutton';
+
+const radiobuttonClassNames = {
+  container: `${baseClassName}_container`,
+  input: `${baseClassName}_input`,
+  label: `${baseClassName}_label`,
+  hintText: `${baseClassName}_hintText`,
+};
+
+type RadiobuttonVariant = 'small' | 'large';
+
+export interface RadiobuttonProps {
+  /** Custom classname to extend or customize */
+  className?: string;
+  /**
+   * Label element content
+   */
+  children?: ReactNode;
+  /** Group multiple Radiobuttons to be inside same selection group */
+  name?: string;
+  /** Unique value to be used inside Radiobutton group for this item */
+  value?: string;
```

Comment:
Why is value optional here?

Reply:
That's a good question! Not sure if there is a use case without value, but current implementation does not work if no value is provided. At least inside a radiogroup nothing is selected if no value is given.

Reply:
Value is now required prop.